src/app/components/zlux-file-explorer/zlux-file-explorer.component.ts

```diff
@@ -116,13 +177,13 @@ export class ZluxFileExplorerComponent implements OnInit, OnDestroy {
   }
 
   hideExplorers() {
-    if (this.ussComponent) {
+    if (this.ussComponent){
```

Comment:
You have a few whitespace changes here, I ignored the first couple but looks like there's quite a few. Generally, we want to avoid changes that only occur by adding spaces, but don't actually change the code. Not only this, it messes with tools like "Git blame" where each line in Git is authored by a committer and little changes like this make it harder to use that tool